README.md

```diff
@@ -1,14 +1,143 @@
+the entry point for all supported SIMP configurations.
 
 ## This is a SIMP module
-This module is a component of the [System Integrity Management Platform](https://github.com/NationalSecurityAgency/SIMP), a compliance-management framework built on Puppet.
+This module is a component of the [System Integrity Management Platform](https://simp-project.com)
 
-If you find any issues, they can be submitted to our [JIRA](https://simp-project.atlassian.net/).
+If you find any issues, please submit them via [JIRA](https://simp-project.atlassian.net/).
 
 Please read our [Contribution Guide](https://simp-project.atlassian.net/wiki/display/SD/Contributing+to+SIMP) and visit our [developer wiki](https://simp-project.atlassian.net/wiki/display/SD/SIMP+Development+Home).
 
-## Work in Progress
+This module should be used within the SIMP ecosystem and will be of limited
+independent use
+
+## Module Description
+
+This module provides a convenient entry point for setting up systems to meet
+the goals of the SIMP Project.
+
+It is effectively a highly malleable Puppet profile that provides mechanisms
+for direct overall system modification and management.
+
+## Setup
+
+### What simp affects
+
+The ``simp`` module is meant to be the central controller of all node
+configurations. The suggested usage is to place the following in your
+environment's ``site.pp``:
+
+```ruby
+include 'simp_options'
+include 'simp'
+```
+
+*NOTE:* If using Puppet Enterprise, you can add the ``simp_options`` and
+``simp`` classes to nodes via the classification interface. Do be sure to
+include ``simp_options`` *before* ``simp`` so that the ``simp`` module has
+appropriate access to the parameters in ``simp_options``.
+
+## Usage
+
+### Basic Usage
+
+It is recommended that you start with one of the SIMP scenarios described below.
+
+These may be set via the ``simp::scenario`` parameter via Hiera.
+
+You may want to tweak individual module settings and should reference the
+[module documentation](https://github.com/simp/pupmod-simp-simp/docs/index.html)
+for full details.
+
+#### SIMP Scenarios
+
+The SIMP module has the following scenarios defined for getting started with
+different configurations easily:
+
+* ``simp``
+  * The default scenario. Enables all modules to support the default SIMP
+    infrastructure configured around security best practices and compatibility
+    with supported security policies as defined in the
+    ``compliance_markup`` module.
+
+* ``simp_lite``
+  * The ``simp`` profile with some of the more aggressive security support
+    modules disabled. These include, but are not limited to, ``iptables``,
+    ``fips``, and ``svckill``.
+
+* ``standalone``
+  * Applies all of the settings in the ``simp`` profile and, after a successful
+    run, either disables ``puppet`` from running again or removes it from the
+    system completely. Has options to ensure that there is a way to get back
+    into the system afterwards.
```

Comment:
The server is left alone in all scenarios. I'll update the docs to make it more clear.